docs/Customization/Memory-Hierarchy.rst

```diff
@@ -44,12 +50,23 @@ you can only use a single core and you cannot give the design an external DRAM.
 
 .. code-block:: scala
 
-    import freechips.rocketchip.subsystem.{WithNoMemPort, WithScratchpadsOnly}
+    import freechips.rocketchip.subsystem.{WithNSmallCore, WithNoMemPort, WithScratchpadsOnly}
```

Comment:
Since the configs in this code block explicitly use the package path for their mixins (new freechips.rocketchip.subsystem.XYZMixin...`), you can delete this line

Reply:
If you don't delete this line then you should have it be `WithNSmallCores`